planner-front/src/Components/Home.js

```diff
@@ -38,17 +40,22 @@ export default function Home() {
                     </Grid>
                     <Grid item xs={3}>
                         <div className="container">
-                            <h2>Saldo: {initialState}</h2>
+                            <BalanceField></BalanceField>
```

Comment:
The convention says that elements that have no content inside should be single elements - like <BalanceField /> insted of <BalanceField></BalanceField>